### PR TITLE
Fix .env not loading in build.rs, use dev.toml

### PR DESCRIPTION
**Pull request by TheButlah**

### CODE REVIEW

File: firmware/Cargo.toml [hunk 1]
```diff
@@ -205,8 +205,8 @@ fugit = "0.3"
 firmware_protocol = { path = "../networking/firmware_protocol", features = [
   "nalgebra031",
 ] }
-load-dotenv = "0.1"
 paste = "1.0"
+load-dotenv = "0.1"
```
Comment (ImUrX) [inline firmware/Cargo.toml:209]: why not use [``dotenvy_macro``](https://crates.io/crates/dotenvy_macro)?

**Review by ImUrX — CHANGES_REQUESTED** [review]

File: firmware/Cargo.toml (hunk 1 above)
Comment (TheButlah) [reply firmware/Cargo.toml:209]: I believe it requires you to specify the env var you want to check, but we actually want to load *all* env vars from .env

**Comment by ImUrX** [comment]
it still doesnt build tho

**Comment by TheButlah** [comment]
will fix tn

File: firmware/.gitignore [hunk 2]
```diff
@@ -3,3 +3,5 @@
 *.bin
 *.uf2
 \.env
+# Convenient for local development. But should not be comitted.
```
Comment (TheButlah) [inline firmware/.gitignore]:
```suggestion
# Convenient for local development. But should not be committed.
```

**Review by ImUrX — APPROVED** [review]

**Pull request merged**